cmd/explaintest/r/explain_complex.result

```diff
@@ -246,7 +246,7 @@ id	estRows	task	operator info
 Sort_10	1.00	root	test.org_department.left_value:asc
 └─HashAgg_15	1.00	root	group by:test.org_department.id, funcs:firstrow(test.org_department.id)->test.org_department.id, funcs:firstrow(test.org_department.ctx)->test.org_department.ctx, funcs:firstrow(test.org_department.name)->test.org_department.name, funcs:firstrow(test.org_department.left_value)->test.org_department.left_value, funcs:firstrow(test.org_department.right_value)->test.org_department.right_value, funcs:firstrow(test.org_department.depth)->test.org_department.depth, funcs:firstrow(test.org_department.leader_id)->test.org_department.leader_id, funcs:firstrow(test.org_department.status)->test.org_department.status, funcs:firstrow(test.org_department.created_on)->test.org_department.created_on, funcs:firstrow(test.org_department.updated_on)->test.org_department.updated_on
   └─Selection_22	0.01	root	or(eq(test.org_employee_position.user_id, 62), or(eq(test.org_department.id, 20), eq(test.org_department.id, 20)))
-    └─HashLeftJoin_24	0.02	root	left outer join, inner:IndexMergeJoin_40 (REVERSED), equal:[eq(test.org_position.id, test.org_employee_position.position_id)]
+    └─HashLeftJoin_24	0.02	root	left outer join, inner:TableReader_72, equal:[eq(test.org_position.id, test.org_employee_position.position_id)]
```

Comment:
we don't need to print which is the inner side child anymore since we've print the `build` and `probe` information in the `id` of its children.

Reply:
`Inner`/`Outer` is the concept for outer join, like _b_ is the inner when _a_ left outer join _b_ , while `build`/`probe` is the concept for hash table.
Is it ok to abandon this information?
It's ok to me but I think someone may have other opinions.

Reply:
Removing `inner` is ok. 
Is it better to keep `REVERSED` so that to point out the optimization we made?